functions/__z.fish

```diff
@@ -164,9 +164,13 @@ function __z -d "Jump to a recent directory."
         if set -q _flag_echo
             printf "%s\n" "$target"
         else if set -q _flag_directory
-            type -q xdg-open;and xdg-open "$target"; and return $status;
-            type -q open;and open "$target"; and return $status;
-            echo "Not sure how to open file manager"; and return 1;
+            if test "$OS" = Windows_NT
+                type -q explorer;and explorer "$target"; return 0;
```

Comment:
let's also return $status for consistency.

Reply:
Basically, it should return `$status`, but I found that in `msys2`, `explorer` always returns 1 even it is executed successfully. And `explorer` is the basic component of Windows. That's why I manually make it to return 0. However, we can still let it return `$status` if you want.

Reply:
> explorer always returns 1 even it is executed successfully

I see. I think it depends on how windows treat return codes. Why isn't 0 the success code :thinking: In that case I think the current behaviour is fine